dockerfiles/buster/Dockerfile.saithrift-server

```diff
@@ -0,0 +1,74 @@
+FROM debian:buster-slim
+
+RUN echo "deb [arch=amd64] http://debian-archive.trafficmanager.net/debian/ buster main contrib non-free" >> /etc/apt/sources.list && \
+        echo "deb-src [arch=amd64] http://debian-archive.trafficmanager.net/debian/ buster main contrib non-free" >> /etc/apt/sources.list && \
+        echo "deb [arch=amd64] http://debian-archive.trafficmanager.net/debian-security/ buster/updates main contrib non-free" >> /etc/apt/sources.list && \
+        echo "deb-src [arch=amd64] http://debian-archive.trafficmanager.net/debian-security/ buster/updates main contrib non-free" >> /etc/apt/sources.list && \
+        echo "deb [arch=amd64] http://debian-archive.trafficmanager.net/debian buster-backports main" >> /etc/apt/sources.list
+
+## Make apt-get non-interactive
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY sai.env /
+
+# Install generic packages
+RUN apt-get -o Acquire::Check-Valid-Until=false update && apt-get install -y \
+        apt-utils \
+        vim \
+        curl \
+        wget \
+        iproute2 \
+        unzip \
+        git \
+        procps \
+        build-essential \
+        graphviz \
+        doxygen \
+        aspell \
+        python3-pip \
+        rsyslog \
+        supervisor
+
+# Add support for supervisord to handle startup dependencies
+RUN pip3 install supervisord-dependent-startup==1.4.0
+
+# Install generic packages
+RUN apt-get install -y \
+        libtemplate-perl \
+        libconst-fast-perl \
+        libmoosex-aliases-perl \
+        libnamespace-autoclean-perl \
+        libgetopt-long-descriptive-perl \
+        aspell-en bison flex g++ \
+        libboost-all-dev libevent-dev libssl-dev \
+        libpython3-dev libpcap-dev
+
+WORKDIR /sai
+
+RUN apt-get install -y thrift-compiler libthrift-dev libthrift-0.11.0 \
+    && pip3 install ctypesgen thrift==0.11.0 ;
+
+ENV SAITHRIFTV2=y
+ENV platform=vs
```

Comment:
In case we set `platform=vs`, I'd expect the build will fail due to a dependency on `saivs`. See https://github.com/opencomputeproject/SAI/blob/a94bbbe43242a4d9e1a4d9f70780ea9251127f5d/test/saithriftv2/Makefile#L41